client/src/ui/BitbakeCommands.ts

```diff
@@ -188,6 +190,26 @@ async function runTaskCommand (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjec
   }
 }
 
+async function startToasterInBrowser (bitbakeDriver: BitbakeDriver): Promise<void> {
+  const DEFAULT_TOASTER_PORT = 8000
+  const command = 'nohup bash -c "source toaster start"'
+  const process = await runBitbakeTerminalCustomCommand(bitbakeDriver, command, 'Toaster')
+  process.onExit(() => {
+    const url = `http://localhost:${DEFAULT_TOASTER_PORT}`
+    void vscode.env.openExternal(vscode.Uri.parse(url)).then(success => {
+      if (!success) {
+        void vscode.window.showErrorMessage(`Failed to open URL ${url}`)
+      }
+    })
+  })
+}
+
+async function stopToaster (bitbakeDriver: BitbakeDriver): Promise<void> {
+  const command = 'source toaster stop'
+  const script = 'source oe-init-build-env && ' + command
+  await runBitbakeTerminalScript(command, bitbakeDriver, 'Toaster', script, true)
```

Comment:
You could use `runBitbakeTerminalCustomCommand` here too, so it would use the env script defined in the user's settings

Reply:
Cool!

Reply:
done